src/components/panels/collapsible-panel/collapsible-panel.js

```diff
@@ -81,32 +82,40 @@ export default class CollapsiblePanel extends React.PureComponent {
       >
         {({ isOpen, toggle, containerStyles, registerContentNode }) => (
           <div
-            className={classnames(this.props.className, {
-              [styles['container-condensed']]: this.props.condensed,
-              [styles['container-open']]: isOpen,
-              [styles['container-theme-light']]: this.props.theme === 'light',
-              [styles['container-theme-dark']]: this.props.theme === 'dark',
-            })}
+            className={classnames(
+              this.props.className,
+              styles[`container-theme-${this.props.theme}`],
```

Comment:
A similar classname was built up by convention elsewhere in the file. Doing the same here shortens the `classnames` "block".